example/rlottiePlayer/rlottie_capi.h

```diff
@@ -0,0 +1,282 @@
+/*
```

Comment:
Please don't attach rlottie_capi.h
rlottie already has it rlottie/inc folder.
This project could use it instead of the copy.
Please check your visual studio project properties to set up with it.